<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IrdlEditorStepSettings</class>
 <widget class="QWidget" name="IrdlEditorStepSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>488</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="Settings">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QScrollArea { background: transparent; }
QScrollArea &gt; QWidget &gt; QWidget { background: transparent; }</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>367</width>
           <height>663</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="StatementStep">
            <property name="title">
             <string>Statement step</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="StatementColorLabel">
               <property name="text">
                <string>Color</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="Utils::QtColorButton" name="StatementColor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="StatementTextcolorLabel">
               <property name="text">
                <string>Textcolor</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="Utils::QtColorButton" name="StatementTextcolor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="TimerStep">
            <property name="title">
             <string>Timer step</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="2" column="0">
              <widget class="QLabel" name="TimerTextcolorLabel">
               <property name="text">
                <string>Textcolor</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="Utils::QtColorButton" name="TimerTextcolor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="TimerColorLabel">
               <property name="text">
                <string>Color</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="Utils::QtColorButton" name="TimerColor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="TimeoutStep">
            <property name="title">
             <string>Timeout step</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="2" column="0">
              <widget class="QLabel" name="TimeoutTextcolorLabel">
               <property name="text">
                <string>Textcolor</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="Utils::QtColorButton" name="TimeoutTextcolor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="TimeoutColorLabel">
               <property name="text">
                <string>Color</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="Utils::QtColorButton" name="TimeoutColor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="SleeptimerStep">
            <property name="title">
             <string>Sleeptimer step</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="2" column="0">
              <widget class="QLabel" name="SleeptimerColorLabel">
               <property name="text">
                <string>Color</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="Utils::QtColorButton" name="SleeptimerTextcolor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="SleeptimerTextcolorLabel">
               <property name="text">
                <string>Textcolor</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="Utils::QtColorButton" name="SleeptimerColor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="MessageStep">
            <property name="title">
             <string>Message step</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="2" column="1">
              <widget class="Utils::QtColorButton" name="MessageTextcolor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="MessageColorLabel">
               <property name="text">
                <string>Color</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="MessageTextcolorLabel">
               <property name="text">
                <string>Textcolor</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="Utils::QtColorButton" name="MessageColor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="CheckmessageStep">
            <property name="title">
             <string>Checkmessage step</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="1" column="1">
              <widget class="Utils::QtColorButton" name="CheckmessageTextcolor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="CheckmessageTextcolorLabel">
               <property name="text">
                <string>Textcolor</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="CheckmessageColorLabel">
               <property name="text">
                <string>Color</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="Utils::QtColorButton" name="CheckmessageColor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="FragmentStep">
            <property name="title">
             <string>Fragment step</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="2" column="0">
              <widget class="QLabel" name="RegionColorLabel">
               <property name="text">
                <string>Regioncolor</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="Utils::QtColorButton" name="FragmentTextcolor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="FragmentTextcolorLabel">
               <property name="text">
                <string>Textcolor</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="Utils::QtColorButton" name="FragmentColor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="FragmentColorLabel">
               <property name="text">
                <string>Color</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="RegionTextcolorLabel">
               <property name="text">
                <string>Regiontextcolor</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="Utils::QtColorButton" name="RegionColor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="Utils::QtColorButton" name="RegionTextcolor">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="Preview">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="PreviewLayout"/>
      </item>
      <item>
       <widget class="QPushButton" name="refresh">
        <property name="text">
         <string>Refresh</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::QtColorButton</class>
   <extends>QToolButton</extends>
   <header location="global">utils/qtcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
